server/intelligence-service/app/mentor/router.py

```
@@ -0,0 +1,70 @@
+
+from app.logger import logger
+from app.mentor.response import generate_response
+from app.mentor.converter import convert_to_langchain_messages
+from app.mentor.models import StreamPart, UIMessage
+from app.mentor.streaming import StreamGenerator
+
+
+router = APIRouter(prefix="/mentor", tags=["mentor"])
+
+
+class ChatRequest(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)